react/components/FilterNavigatorTitleTag.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { FormattedMessage } from 'react-intl'
+
+const CSS_HANDLES = ['filterMessage']
+
+const FilterNavigatorTitleTags = ({ filtersTitleHtmlTag }) => {
```

Comment:
You can add a default value of `h5` here and remove the `if`
```suggestion
const FilterNavigatorTitleTags = ({ filtersTitleHtmlTag = 'h5' }) => {
```

Reply:
Fixed